### ISSUE TITLE
Results on Cityscapes with ResNet101 under 1/16 setting are lower than reported

### ISSUE THREAD

**Issue by QiaolingXiong** [cvlab-stonybrook/Beyond-Pixels#7]
Hello, I reproduced the results of the 1/16 setting on the Cityscapes dataset using ResNet101 as the backbone, but I only achieved 76.58 mIoU, which is significantly lower than the 78.5 mIoU reported in the paper. Could you help me understand why there is such a gap? Are there any specific configurations or tricks that I might have missed during reproduction?  

Thank you in advance!

**Comment by prantikhowlader** [comment]
Hi 

For 1/16 setting for cityscapes, we reduce the learning rate, further the weight of multi label classifier training is reduced after 50 epochs. To prevent the classifier from overfitting to the data

**Comment by haruishi43** [comment]
@prantikhowlader Hey Prantik, could you share your exact hyperparameters and the code you used (loss weight annealing) to get that 78.5 score on Cityscapes using DLV3+? That’s pretty impressive, and I’d love to try it out myself.
@QiaolingXiong Did you happen to reproduce their results?
